### ISSUE TITLE
Old site

### ISSUE THREAD

**Issue by mcfiredrill** [pineapple-tracker/pineapple-tracker.github.io#1]
The old site from Grace was really cool. I guess the source is lost though?

**Comment by nikolas** [comment]
I know I wish we could find it. it was like http://grace.evergreen.edu/~milton13/tracker.php or something. Maybe we could talk to isaac or peter and see if there's an archive of your grace home directory somewhere.

**Comment by peterquest** [comment]
10:00 @p_q `nik`: looking... don't hold your breath

**Comment by peterquest** [comment]
yeah no dice. home dirs get archived after accounts get removed from AD and then they get nuked after 6 months or so. time to rewrite!

**Comment by mcfiredrill** [comment]
Damn, :sweat: 
thanks for looking @peterquest !

**Comment by nikolas** [comment]
well i'll try and make this one look like the old one from what i can remember. currently it doesn't look anything like it.
- [ ] needs to be a deeper shade of purple
- [ ] what else? do you have a photographic memory?

**Comment by mcfiredrill** [comment]
@nikolas 
The purple was #66066. I remember I memorized that color for some reason.
I remember it said pineapple tracker with text generated from `toilet` using the --gay flag.
Also I remember using `include` from PHP.

I think you put transparent background image up later. The only color was black. I think it was a picture of some graffitti you saw behind the bowling alley or something.

I think that was basically it. Maybe a small description or a copy of the readme was there. I can't remember if there were any links.

**Comment by nikolas** [comment]
Oh yeah the graffiti was this pic messed with in imagemagick: https://www.flickr.com/photos/18873310@N00/3265047472/in/pool-666503@N21/

I'll find the right file or at least make something like it. I remember using php too so we could use `include` lol. So that means there was probably a few pages, like maybe an `about.php`, `contact.php`?
